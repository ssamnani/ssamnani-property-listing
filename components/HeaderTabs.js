import React from "react";
import "../styles/HeaderTabs.module.css";

function HeaderTabs() {
  return (
    <div className="headerTabs">
      <ul>
        <li>
          <a className="active">Buy</a>
        </li>
        <li>
          <a>Rent</a>
        </li>
        <li>
          <a>New Launches</a>
        </li>
        <li>
          <a>Advertise</a>
        </li>
        <li>
          <a>News & Lifestyle</a>
        </li>
        <li>
          <a>Events</a>
        </li>
        <li>
          <a>International</a>
        </li>
      </ul>
    </div>
  );
}

export default HeaderTabs;
