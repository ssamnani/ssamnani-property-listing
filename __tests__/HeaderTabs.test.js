import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import HeaderTabs from "../components/HeaderTabs";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render("<HeaderTabs />", div);
});

// Snapshot Testing
it("matches snapshot", () => {
  const tree = renderer.create(<HeaderTabs />).toJSON();
  expect(tree).toMatchSnapshot();
});
