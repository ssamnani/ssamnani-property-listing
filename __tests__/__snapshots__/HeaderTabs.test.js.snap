// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="headerTabs"
>
  <ul>
    <li>
      <a
        className="active"
      >
        Buy
      </a>
    </li>
    <li>
      <a>
        Rent
      </a>
    </li>
    <li>
      <a>
        New Launches
      </a>
    </li>
    <li>
      <a>
        Advertise
      </a>
    </li>
    <li>
      <a>
        News & Lifestyle
      </a>
    </li>
    <li>
      <a>
        Events
      </a>
    </li>
    <li>
      <a>
        International
      </a>
    </li>
  </ul>
</div>
`;
